spec/locale.html

```diff
@@ -0,0 +1,431 @@
+<emu-clause id="locale-objects">
+  <h1>Locale Objects</h1>
+
+  <emu-clause id="sec-intl-locale-constructor">
+    <h1>The Intl.Locale Constructor</h1>
+
+    <p>
+      The Locale constructor is a standard built-in property of the Intl object.
+    </p>
+
+    <emu-clause id="sec-apply-options-to-tag" aoid=ApplyOptionsToTag>
+      <h1>ApplyOptionsToTag( _tag_, _options_ )</h1>
+      <p>
+        The following algorithm refers to <a href="https://www.unicode.org/reports/tr35/#Identifiers">UTS 35's Unicode Language and Locale Identifiers grammar</a>.
+      </p>
+
+      <emu-alg>
+        1. Assert: Type(_tag_) is String.
+        1. If IsStructurallyValidLanguageTag(_tag_) is *false*, throw a *RangeError* exception.
+        1. Let _language_ be ? GetOption(_options_, `"language"`, `"string"`, *undefined*, *undefined*).
+        1. If _language_ is not *undefined*, then
+          1. If _language_ does not match the `unicode_language_subtag` production of the `unicode_language_id`, throw a *RangeError* exception.
+        1. Let _script_ be ? GetOption(_options_, `"script"`, `"string"`, *undefined*, *undefined*).
+        1. If _script_ is not *undefined*, then
+          1. If _script_ does not match the `unicode_script_subtag` production of the `unicode_language_id`, throw a *RangeError* exception.
+        1. Let _region_ be ? GetOption(_options_, `"region"`, `"string"`, *undefined*, *undefined*).
+        1. If _region_ is not *undefined*, then
+          1. If _region_ does not match the `unicode_region_subtag` production of the `unicode_language_id`, throw a *RangeError* exception.
+        1. Set _tag_ to CanonicalizeUnicodeLocaleId(_tag_).
+        1. If _language_ is not *undefined*,
+          1. Assert: _tag_ matches the `unicode_locale_id` production.
+          1. Set _tag_ to _tag_ with the substring corresponding to the `unicode_language_subtag` production replaced by the string _language_.
+        1. If _script_ is not *undefined*, then
+          1. If _tag_ does not contain a `unicode_script_subtag` production, then
+            1. Set _tag_ to the concatenation of the `unicode_language_subtag` production of _tag_, `"-"`, _script_, and the rest of _tag_.
+          1. Else,
+            1. Set _tag_ to _tag_ with the substring corresponding to the `unicode_script_subtag` production replaced by the string _script_.
+        1. If _region_ is not *undefined*, then
+          1. If _tag_ does not contain a `unicode_region_subtag` production, then
+            1. Set _tag_ to the concatenation of the `unicode_language_subtag` production of _tag_, the substring corresponding to the `"-" unicode_script_subtag` production if present, `"-"`, _region_, and the rest of _tag_.
```

Comment:
The find+related option for string literal formatting will only break on this line. You should update this line first:

```
             1. Set _tag_ to the concatenation of the `unicode_language_subtag` production of _tag_, the substring corresponding to `"-"` and the `unicode_script_subtag` production if present, `"-"`, _region_, and the rest of _tag_.
```